packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -215,10 +215,10 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
                   right: `-${pxToRem(primaryBorderWidth * 2)}`,
                   bottom: `-${pxToRem(primaryBorderWidth * 2)}`,
                   left: `-${pxToRem(primaryBorderWidth * 2)}`,
-                  borderWidth: `${pxToRem(primaryBorderWidth)}`,
+                  borderWidth: pxToRem(primaryBorderWidth),
                   borderStyle: 'solid',
-                  borderColor: `${primaryBorderColorFocusIndicator}`,
-                  borderRadius: '3px',
+                  borderColor: primaryBorderColorFocusIndicator,
```

Comment:
we might think about making names consistent here as well - as it seems that `focusIndicator` largely states for reflecting 'focus' state aspects. Maybe we could rename it to `primaryBorderColorFocused`?

Reply:
I agree, but should note that it would mean introducing breaking changes, just because of refactoring? Are we ok with this?

Reply:
no, surely - in that case lets better wait with them. Thank you!